caniusepython3/dependencies.py

```diff
@@ -97,7 +98,9 @@ def blocking_dependencies(projects, py3_projects):
                     # can't port.
                     del reasons[parent]
                     continue
+                log.info('Dependencies of {0}: {1}'.format(project, deps))
```

Comment:
@msabramo  Hmm, maybe I'm mistaken but I think this should have been logging parent, not project, isn't it..? ([PR224](https://github.com/brettcannon/caniusepython3/pull/224)).  Sorry for the half-decade ping!